tests/HomeworkCalculatorTesting.spec.js

```diff
@@ -0,0 +1,82 @@
+const { test, expect } = require('@playwright/test');
+/*const { BasicCalculatorPage } = require('../pages-Calculator/basicCalculatorPage');
+
+
+test.describe('', () => {
+    let page;
+    
+    test.beforeAll(async ({ browser }) => {
+      page = await browser.newPage();
+      startPage = new BasicCalculatorPage(page);
+
+        });
+   test.beforeEach(async () => {
+      await startPage.goto();
+    });
+  });*/
+       
+ 
+test('Testing page to be opened', async ({ page }) => {
+  await page.goto('https://testsheepnz.github.io/BasicCalculator');
+  const calculatorPageTop = await page.isVisible('#page-top');
+  expect(calculatorPageTop).toBe(true);
+  
+});
+
+test('Check that calculator is working', async ({ page }) => {
+  await page.goto('https://testsheepnz.github.io/BasicCalculator');
+  await page.waitForSelector('#page-top');
+  await page.fill('#number1Field', '5');
+  await page.fill('#number2Field', '3');
+  await page.click('#calculateButton');
+  const result2Content = await page.isVisible('#numberAnswerField');
+  expect(result2Content).toContainEqual('8');
+  
+    });
+
+    test('Check that calculator adds numbers', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
+      await page.waitForSelector('#page-top');
+      await page.selectOption('select#selectBuild', '0');
+      await page.fill('#number1Field', '5');
+      await page.fill('#number2Field', '3');
+      await page.click('#selectOperationDropdown');
+      await page.selectOption('select#selectOperationDropdown','Add');
+      await page.click('#calculateButton');
+      const result1Content = await page.textContent('#numberAnswerField');
+      expect(result1Content).toContain('8');
+     
+    });
+      
+
+
+          
+  const inputLengths = ['8'];
+  inputLengths.forEach(inputLengths => {
+  test(`Fill ${inputLengths} characters`, async ({ page }) => {
```

Comment:
not sure I understand the test name and the test itself. Does it mean to test that user is able to input specific number?